lib/p2p/Pool.ts

```diff
@@ -93,7 +93,7 @@ class Pool extends EventEmitter {
   }
 
   public broadcastOrder = (order: orders.OutgoingOrder) => {
-    const orderPacket = new OrderPacket().fromOutgoingOrder(order);
```

Comment:
if you don't want to use this factory method, remove it from the class as well (unless you wanted to keep it as an example).

Reply:
Thanks, it was just an oversight.